client/post-editor/editor-drawer-well/style.scss

```diff
@@ -49,14 +53,13 @@
 	cursor: pointer;
 }
 
-.editor-drawer-well__remove-icon {
+.editor-drawer-well__remove .editor-drawer-well__remove-icon.gridicon {
 	position: absolute;
 		top: 50%;
-		right: 0;
-		left: 0;
-	transform: translateY( -50% );
-	font-size: 14px;
-	font-weight: bold;
-	text-align: center;
+		left: 50%;
```

Comment:
>Extra tab?

@gwwar I must have overlooked this comment the first time around. Extra tab is an exception for positioning attributes ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#positioning)).